### PR TITLE
Add cactus.cards to allowlist

### LINKED ISSUE #5887: [Legitimate Site Blocked] cactus.cards
https://cactus.cards/

### PR DESCRIPTION
**Pull request by trn1ty**
Fixes #5887.

### CODE REVIEW

**Comment by davidwgrossman** [comment]
@devenblake in the future, would you please condense these changes into 1 issue? This will probably mean 10x less work for you and 25x less work for me :) 

The biggest issue is that I have to resolve conflicts, wait for tests to pass again, and then merge after I merge every PR.

**Comment by devenblake** [comment]
Tests failing after failed merge. I don't really understand GitHub so it beats me what the core issue is but I'll fix the symptom real quick.

**Comment by devenblake** [comment]
> @devenblake in the future, would you please condense these changes into 1 issue? This will probably mean 10x less work for you and 25x less work for me :)
> 
> The biggest issue is that I have to resolve conflicts, wait for tests to pass again, and then merge after I merge every PR.

[Please e-mail me](http://www.trinity.moe/#contact). I've a couple questions about how to move forward with batching changes in the way you describe.

**Review by gravelcycles — APPROVED** [review]

**Pull request merged**
